src/database/sqlite/mod.rs

```diff
@@ -260,6 +260,17 @@ impl SqliteConn {
         .expect("Db must not fail")
     }
 
+    /// List coins that are being spent and that are waiting confirmation.
```

Comment:
nit: "List coins that are being spent and whose spending transaction is still unconfirmed"